### PR TITLE
:sparkles: feat: Added Login Integration

### PR DESCRIPTION
**Pull request by softdevandre**
## Task

This pull request refers to task LOR-188

- [ ] Does not apply to this pull request

## Description

- Implemented login integration with API
- Implemented temporary logout button on Home Screen to prevent user being stuck after credentials save

## Types of changes #

- [x] New feature (non-breaking change which adds functionality).
- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## What changed: #

- [x] UI
- [ ] Business rule
- [ ] Documentation
- [x] Gradle or Build
- [x] Navigation
- [ ] Tests
- [x] Resource files
- [ ] Project architecture
- [ ] Code style

## Checklist #

- [x] Have tested the changes.
- [x] Met all the acceptance requirements of this task.
- [x] Verified if branch is up-to-date with `development` (if not - rebase it or merge it).
- [x] Resolve git conflicts

## If any of the acceptance criteria are not met, please explain why below #
What is the criteria? Why it is different?

### CODE REVIEW

File: feature/auth/login/build.gradle.kts [hunk 1]
```diff
@@ -59,4 +61,7 @@ dependencies {
 
     implementation(project(":core:ui"))
     implementation(project(":core:util"))
+    implementation(project(":core:network"))
+    implementation(project(":data:logbook"))
+    implementation(project(mapOf("path" to ":data:auth")))
```
Comment (hartmannjean) [inline feature/auth/login/build.gradle.kts]: Isso aqui acho que da pra deixar no padrão dos outros

File: feature/home/build.gradle.kts [hunk 2]
```diff
@@ -50,6 +50,11 @@ dependencies {
     implementation(libs.ui.tooling.preview)
     implementation(libs.navigation.compose)
     implementation(libs.lifecycle.runtime.compose)
+    implementation(project(mapOf("path" to ":core:network")))
+    implementation(project(mapOf("path" to ":data:auth")))
```
Comment (hartmannjean) [inline feature/home/build.gradle.kts]: Colocar no mesmo padrão dos outros implementation(project(":data:auth"))

**Review by hartmannjean — APPROVED** [review]
Ok!

File: data/auth/src/main/java/com/loryblu/data/auth/extensions/HttpResponseSignInResultExtension.kt
Comment (OdisBy) [inline data/auth/src/main/java/com/loryblu/data/auth/extensions/HttpResponseSignInResultExtension.kt:1]: This file name can be shorted

**Review by OdisBy — APPROVED** [review]
🎁

File: feature/home/build.gradle.kts (hunk 2 above)
Comment (softdevandre) [reply feature/home/build.gradle.kts]: Done! :+1:

File: feature/auth/login/build.gradle.kts (hunk 1 above)
Comment (softdevandre) [reply feature/auth/login/build.gradle.kts]: Changed :fist_oncoming:

File: data/auth/src/main/java/com/loryblu/data/auth/extensions/HttpResponseSignInResultExtension.kt
Comment (softdevandre) [reply data/auth/src/main/java/com/loryblu/data/auth/extensions/HttpResponseSignInResultExtension.kt:1]: :ok_hand: Resolved

**Pull request merged**